src/main/java/mate/academy/dao/UserDao.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.model.User;
+
+public interface UserDao {
+    User save(User user);
+
+    Optional<User> get(Long id);
```

Comment:
What is it for?

Reply:
this is a redundant method. it is not used

Reply:
i removed it